docs/community_features.md

```diff
@@ -1093,6 +1093,15 @@ to each individual note onset. ([#1978])
 
 - As the UI and implementation is still experimental, a community setting has to be activated to access the `CHORD LIBRARY` keyboard. See the [Community Features Menu](#5-community-features-menu-aka-runtime-settings) for more information.
 
+#### 4.4.1.8 - Display Piano Keyboard Layout
+
+- ([#2983]) Provides a classic piano keyboard layout for Deluge.
+- One screen displays 5 octaves with different colors. But you can scroll this keyboard vertically to increase octaves to the highest or decrease to the lowest. You can also shift the keyboard to start on a different note than C. This is done by using the horizontal scroll.
+- If the Scale mode is enabled, all the notes in this scale will be displayed brighter than the notes outside the scale. Of course, you can change the scale, and the keys will change accordingly. The same thing with the root note (the brightest one).
+- The same notes you play are highlighted in other octaves as well.
+- This keyboard highlights incoming MIDI notes.
+- You can set this keyboard by default in `MENU -> DEFAULTS -> UI -> KEYBOARD -> LAYOUT -> PIAN`
```

Comment:
Can you add an entry for this new menu in our menu hierarchies document?

Here: https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/menu_hierarchies.md

![image](https://github.com/user-attachments/assets/46df7d6e-cf51-41e1-b043-18ebf8e36fbc)

Add the layout Piano (PIAN)

Thx